mgr/Conscript-standard

```diff
@@ -792,6 +792,10 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
     } elsif ( $pkg =~ m/^StJetFinder$/ ){
 	$CXXFLAGS .=  " -I".$env1->{ENV}->{OPTSTAR}."/include";
 
+        if ( defined($ENV{FASTJET_DIR}) ) {
+          $CPPPATH = $ENV{FASTJET_DIR}."/include".$main::PATH_SEPARATOR.$CPPPATH;
+        }
```

Comment:
What about path to the libraries? Also, I wonder if you've considered querying paths using `fastjet-config`.

Reply:
> What about path to the libraries? 

Are you suggesting to link against the fastjet lib? Sure, why not. But I don't really know how to do it properly with cons. Can you make a concrete suggestion?

> Also, I wonder if you've considered querying paths using `fastjet-config`.

No, I just set the path by hand:

https://github.com/star-bnl/star-sw/pull/75/commits/3a4b404c0742a9eefe10b904a4169c682394985e

Reply:
> Are you suggesting to link against the fastjet lib? Sure, why not. But I don't really know how to do it properly with cons. Can you make a concrete suggestion?

I've forgotten that we load those dynamically:
https://github.com/star-bnl/star-sw/blob/ecfe0a03767393e85db5fcb91f9d05077c9747fd/StRoot/StJetMaker/macros/RunJetFinder2012pro.C#L34
I would say there are some benefits to linking directly, but that's outside of the scope of this PR.

> No, I just set the path by hand:
> 
> [3a4b404](https://github.com/star-bnl/star-sw/commit/3a4b404c0742a9eefe10b904a4169c682394985e)

For sure, this is the intention. Was wondering if you think it's a good idea. I guess, it's not that important.